### PR TITLE
HBW-64 Rename config activiti.yml and, swap camunda and activiti func…

### PR DESCRIPTION
**Pull request by mxsmirnoff**
…tions

### CODE REVIEW

File: hbw/app/models/hbw/camunda/adapter.rb [hunk 1]
```diff
@@ -29,11 +29,11 @@ def submit(user_email, entity_class, task_id, form_data)
       end
 
       def get_variables(user, entity_class, entity_code)
-        [
-          { name: :initiator,                    value: user.id,     type: :string },
-          { name: :initiatorEmail,               value: user.email,  type: :string },
-          { name: entity_code_key(entity_class), value: entity_code, type: :string }
-        ]
+        {
+            :initiator                    => { value: user.id,     type: :string },
```
Comment (sugarfree1) [inline hbw/app/models/hbw/camunda/adapter.rb]: Should be 2 spaces, not 4.

File: hbw/app/models/hbw/activiti/adapter.rb [hunk 2]
```diff
@@ -26,11 +26,11 @@ def submit(user_email, entity_class, task_id, form_data)
       end
 
       def get_variables(user, entity_class, entity_code)
-        {
-          :initiator                    => { value: user.id,     type: :string },
-          :initiatorEmail               => { value: user.email,  type: :string },
-          entity_code_key(entity_class) => { value: entity_code, type: :string }
-        }
+        [
+            { name: :initiator,                    value: user.id,     type: :string },
```
Comment (sugarfree1) [inline hbw/app/models/hbw/activiti/adapter.rb]: 2 spaces, not 4.

**Comment by mxsmirnoff** [comment]
Updated

**Comment by mxsmirnoff** [comment]
Updated

**Pull request merged**